packages/@okta/vuepress-site/docs/guides/anything-as-a-source/index.md

```diff
@@ -50,16 +48,16 @@ The Identity Sources API synchronizing data flow uses an [Identity Source Sessio
 ### Identity Source Session status
 
 * **CREATED**: The Identity Source Session object has been created for a specific Custom Identity Source integration, and you can load data to the session at this stage. Data import processing hasn't been invoked, and you can cancel the session at this stage.
-* **TRIGGERED**: Okta is processing the uploaded data in the Identity Source Session. You can’t load new data to the Identity Source Session object at this stage, and you can't cancel the session. You can view sessions with this status in the [Import Monitoring](https://help.okta.com/okta_help.htm?id=ext-view-import-monitoring-dashboard) page from the Admin Console.
-* **COMPLETED**: The data in the Identity Source Session object has been processed by Okta. You can’t upload new data to the Identity Source Session object if it has the `COMPLETED` status. The synchronization data job is considered complete.
+* **TRIGGERED**: Okta is processing the uploaded data in the Identity Source Session. You can't load new data to the Identity Source Session object at this stage, and you can't cancel the session. You can view sessions with this status in the [Import Monitoring](https://help.okta.com/okta_help.htm?id=ext-view-import-monitoring-dashboard) page from the Admin Console.
+* **COMPLETED**: The data in the Identity Source Session object has been processed by Okta. You can't upload new data to the Identity Source Session object if it has the `COMPLETED` status. The synchronization data job is considered complete.
```

Comment:
```suggestion
* **COMPLETED**: The data in the Identity Source Session object has been processed by Okta. You can't upload new data to the Identity Source Session object if it has this status, because the synchronization data job is considered complete.
```